codemods/use-i18n-hook/example-after.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import { translate, Text, useI18n } from 'cozy-ui/transpiled/react'
+
+const ComponentUsingT = () => {
+  const { t } = useI18n()
+
+  return (
+    <Padded>
+      <Title>{t('Title1')}</Title>
+      <Text>{t('Text1')}</Text>
+    </Padded>
+  );
+}
+
+const ComponentUsingTAndF = () => {
+  const { t,f } = useI18n()
+
+  return (
+    <Padded>
+      <Title>{t('Title2')}</Title>
+      <Text>{f(new Date())}</Text>
+    </Padded>
+  );
+}
+
+const EnhancedComponentUsingTAndF = translate()(DumbComponentUsingTAndF)
```

Comment:
Are you sure this line is left after codemod ?

Reply:
The translate() is removed but the line is left.